### PR TITLE
consolidate discover content

### PR DESCRIPTION
**Pull request by imatwawana**
This PR:
- Moves Discover content from Sentry Basics > Best Practices to under Discover Queries
- Makes minor style and wording edits
- Adds redirects
- Updates sidebar order under Discover Queries

**Out of scope**: major edits to this content; these will be done on a separate PR

### CODE REVIEW

File: src/docs/product/discover-queries/uncover-trends.mdx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+- Search condition: `event.type:error`
+- Table columns: `release` (field), `count()`, `count_unique(issue)`
+
+![Issues per release](012.png)
+
+From here you can add a release as a filter and change the columns to `count()` and `title` to see similar errors. Then you can click the "Open Group" icon to drill down to individual events. You can also open the release in **Releases** from the "Results" table:
+
+<!--Add screenshot-->
+
```
Comment (imatwawana) [inline src/docs/product/discover-queries/uncover-trends.mdx:106]:
```suggestion
```

File: src/docs/product/discover-queries/uncover-trends.mdx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+### Errors per Release
+
+To find out how the health of a specific project is improving (or not) over time as you release new versions, create the following query:
+
+- Search condition: `event.type:error`
+- Table columns: `release` (field), `count()`, `count_unique(issue)`
+
+![Issues per release](012.png)
+
+From here you can add a release as a filter and change the columns to `count()` and `title` to see similar errors. Then you can click the "Open Group" icon to drill down to individual events. You can also open the release in **Releases** from the "Results" table:
```
Comment (imatwawana) [inline src/docs/product/discover-queries/uncover-trends.mdx:103]:
```suggestion
From here you can add a release as a filter and change the columns to `count()` and `title` to see similar errors. Then you can click the "Open Group" icon to drill down to individual events. You can also open the release in **Releases** from the "Results" table.
```

**Review by PeloWriter — APPROVED** [review]
Good structural change.

File: src/docs/product/discover-queries/uncover-trends.mdx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+- Search condition: `event.type:error has:stack.filename`
+- Table columns: `stack.filename`, `count()`
+
+![By File Names](010.png)
+
+You can continue exploring a specific filename by adding it to the filter and changing the table columns to display the major error culprits in that file:
+
+![Culprits](008.png)
+
```
Comment (imatwawana) [inline src/docs/product/discover-queries/uncover-trends.mdx:93]: @wmak following our convo yesterday, can you review the Errors per Release section and let me know if it's accurate?

File: src/docs/product/discover-queries/uncover-trends.mdx [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+1. Navigate to **Discover** and click on "Build a new query".
+
+2. In the Global Header, select the projects, environments, and time range of errors you'd like to query.
+
+3. Sentry monitors different types of events for errors and performance. To query issues, filter by `event.type:error` in the search bar.
+
+   ![Filter by Error type](004.png)
+
+4. Edit the "Results" table columns to show the number of events [`count()`], unique issues [`count_unique(issue)`], and impacted users [`count_unique(user)`] per project.
```
Comment (wmak) [inline src/docs/product/discover-queries/uncover-trends.mdx:26]: nit: to match order in the screenshot
```suggestion
4. Edit the "Results" table columns to show the number of events [`count()`], impacted users [`count_unique(user)`], and unique issues [`count_unique(issue)`] per project.
```

File: src/docs/product/discover-queries/uncover-trends.mdx [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+- Modify the table columns to show the crash `message` and `count()`.
+
+  ![Native Crashes](007.png)
+
+### Files With Most Errors
+
+To find out which files in your codebase are generating the most errors, run the following query in Discover:
+
+- Search condition: `event.type:error has:stack.filename`
+- Table columns: `stack.filename`, `count()`
```
Comment (wmak) [inline src/docs/product/discover-queries/uncover-trends.mdx:86]:
```suggestion
- Table columns: `stack.filename`, `count()`, `count_unique(issue)`
```

File: src/docs/product/discover-queries/uncover-trends.mdx [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+![By File Names](010.png)
```
Comment (wmak) [inline src/docs/product/discover-queries/uncover-trends.mdx:88]: Looks like in the screenshot the display dropdown has also been changed to previous period so you can compare the count between the current previous and the last

**Review by wmak — APPROVED** [review]
some minor comments but LGTM

File: src/docs/product/discover-queries/uncover-trends.mdx (hunk 6 above)
Comment (imatwawana) [reply src/docs/product/discover-queries/uncover-trends.mdx:88]: Do you mean between the first and second screenshot under this heading? Or in general? Both screenshots in this section use the previous period display.

File: src/docs/product/discover-queries/uncover-trends.mdx [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+You can continue exploring a specific filename by adding it to the filter and changing the table columns to display the major error culprits in that file:
+
+![Culprits](008.png)
+
+### Errors per Release
+
+To find out how the health of a specific project is improving (or not) over time as you release new versions, create the following query:
+
+- Search condition: `event.type:error`
+- Table columns: `release` (field), `count()`, `count_unique(issue)`
```
Comment (imatwawana) [inline src/docs/product/discover-queries/uncover-trends.mdx:99]:
```suggestion
- Table columns: `release`, `count()`, `count_unique(issue)`
```

File: src/docs/product/discover-queries/uncover-trends.mdx (hunk 6 above)
Comment (wmak) [reply src/docs/product/discover-queries/uncover-trends.mdx:88]: In general, sorry more specifically: I think we should mention changing the dropdown as part of this example

File: src/docs/product/discover-queries/uncover-trends.mdx [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+redirect_from:
+  - /guides/discover/
+  - /product/sentry-basics/guides/discover/
+description: "Learn how to work with the data in Discover to gain useful insights into the health and stability of your applications."
+---
+
+Discover is a powerful query engine that allows you to query all your error metadata across projects and applications. As you've probably noticed, the Sentry SDKs installed in your applications capture huge amounts of events, each enriched with contextual data about the underlying error, platform, device, user, and more. We also highly encourage enriching your error data through the SDK by adding custom tags and configuring the release and environment. For more information see [Put your Data to Work](/product/sentry-basics/guides/enrich-data/).
+
+Let's go over some examples of how to work with all this data to gain some useful insights into the health and stability of your applications.
+
```
Comment (imatwawana) [inline src/docs/product/discover-queries/uncover-trends.mdx:13]:
```suggestion

<Note>

Some table column options may be repeated if you’ve set up a tag with the same name as a Sentry field. For the examples that follow, use the table column option marked “field.” 

</Note>

```

File: src/docs/product/discover-queries/uncover-trends.mdx [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+### Files With Most Errors
+
+To find out which files in your codebase are generating the most errors, run the following query in Discover:
+
+- Search condition: `event.type:error has:stack.filename`
+- Table columns: `stack.filename`, `count()`, `count_unique(issue)`
+
+![By File Names](010.png)
+
```
Comment (imatwawana) [inline src/docs/product/discover-queries/uncover-trends.mdx:95]:
```suggestion

Change the "Display" selection to compare the current period to the previous one, for example.
 
```

Comment (wmak) [reply src/docs/product/discover-queries/uncover-trends.mdx:95]: hmmm I think cause its in the screenshot we don't need the `for example.` part

**Pull request merged**